pkgs/by-name/ch/changelogging/package.nix

```
@@ -0,0 +1,34 @@
+{
+  lib,
+  fetchCrate,
+  rustPlatform,
+  versionCheckHook,
+  nix-update-script,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "changelogging";
+  version = "0.7.0";
+
+  src = fetchCrate {
+    inherit pname version;
+    hash = "sha256-orTUCBHacD0MQNfhOUWdh9RxT/9YNvgfCHFDr2eNQic=";
+  };
+
+  cargoHash = "sha256-aV0Jf/szJq4x6590WPLG8ctB7CR6k+yiR69GcEcbPhI=";
```

```suggestion
  useFetchCargoVendor = true;
  cargoHash = "";
```
We are moving to the new cargo fetcher nixpkgs-wide. Please build again to get the correct hash.